packages/android_alarm_manager/android/build.gradle

```diff
@@ -1,3 +1,16 @@
+def PLUGIN = "android_alarm_manager";
+def ANDROIDX_WARNING = "flutterPluginsAndroidXWarning";
+gradle.buildFinished { buildResult ->
+    if (buildResult.failure && !rootProject.ext.has(ANDROIDX_WARNING)) {
+        println '         *********************************************************'
+        println 'WARNING: This version of ' + PLUGIN + ' will break your Android build if it isn\'t compatible with AndroidX.'
```

Comment:
`will break your Android build if your project or its dependencies aren't compatible with AndroidX.` (not sure if the `or its dependencies` part is true - do you know?)

Reply:
I'm not sure if it's universally true, but I've managed to crash the build at least once from this. Added.